services/grip/frontend/src/components/Contacts.vue

```diff
@@ -0,0 +1,105 @@
+<script setup lang="ts">
+import { inject, ref } from "vue";
+import Fieldset from "primevue/fieldset";
+import DataTable from "primevue/datatable";
+import Column from "primevue/column";
+import ProgressSpinner from "primevue/progressspinner";
+import axios from "axios";
+import Button from "primevue/button";
+import Dialog from "primevue/dialog";
+import CreateContact from "./CreateContact.vue";
```

Comment:
Not important but would be nice to group the primevue imports in order together

Reply:
Agreed... that would be much cleaner. I'll check to see if Rollup can handle that.